spec/yaml/swiftnav/sbp/deprecated.yaml

```diff
@@ -16,6 +16,22 @@ include:
   - types.yaml
 definitions:
 
+ - MSG_BOOTLOADER_HANDSHAKE_DEPRECATED:
+    id: 0x00B0
+    public: True
```

Comment:
Yeah, maybe they should inherit their previous state? Like if they were public, they stay public? And vice versa.